### PR TITLE
Support tooltips in group headers

### PR DESCRIPTION
**Pull request by smbea**
Test against [this `bpmn-js-properties-panel` branch](https://github.com/bpmn-io/bpmn-js-properties-panel/tree/tooltips)

### CODE REVIEW

File: src/components/entries/Tooltip.js [hunk 1]
```diff
@@ -107,7 +107,9 @@ function Tooltip(props) {
       onMouseEnter={ showTooltip }
       onMouseLeave={ ()=> clearTimeout(timeout) }
       onFocus={ showTooltip }
-      onKeyDown={ hideTooltipViaEscape }>
+      onKeyDown={ hideTooltipViaEscape }
+      onMouseDown={ (e)=> {e.preventDefault();} }
```
Comment (smbea) [inline src/components/entries/Tooltip.js:111]: prevents tooltip from staying open after opening to click group

Comment (philippfromme) [reply src/components/entries/Tooltip.js:111]: Doesn't seem to work for me.

brave_DS1UsEqP8r

Comment (philippfromme) [reply src/components/entries/Tooltip.js:111]: Resolved ✅

**Review by philippfromme — APPROVED** [review]
To the moon! 🚀 🌚

**Comment by philippfromme** [comment]
Let's go ahead and merge this.

**Pull request merged**